sql/rpl_slave.cc

```diff
@@ -6691,6 +6691,7 @@ static int slave_start_workers(Relay_log_info *rli, ulong n, bool *mts_inited) {
    Coordinator finalizes with its MTS running status to reset few objects.
 */
 static void slave_stop_workers(Relay_log_info *rli, bool *mts_inited) {
+  DBUG_TRACE;
```

Comment:
Same comment as above: as this DBUG_TRACE doesn't exists in PS, we have to add it inside WITH_WSREP if we need it.